scripts/babel-relay-plugin/src/babelAdapter.js

```diff
@@ -26,7 +26,25 @@ function babelAdapter(
 ): mixed {
   if (Plugin == null) {
     // Babel 6.
-    return visitorsBuilder(t);
+    const { visitor: { Program, TaggedTemplateExpression } } = visitorsBuilder(t);
+
+    const taggedTemplateExpressionVisitor = {
+      TaggedTemplateExpression(path) {
+        return TaggedTemplateExpression(path, this);
+      }
+    }
+
+    /**
+     * Run both transforms on Program, to make sure that they run before other plugins
+     */
+    return {
+      visitor: {
+        Program(path, state) {
+          Program(path, state);
```

Comment:
I'm not a super Babel expert, and better safe than sorry. Does the return value never matter?